public/static/app/perf/TrendSamples.js

```diff
@@ -38,8 +38,8 @@ mciModule.factory('TrendSamples', function() {
           this.seriesByName[rec.name] = [];
         }
 
-        var maxThreadOpsPerSecItem = _.max(rec.results, function(d) {
-          return d.ops_per_sec
+        var maxValues = _.max(rec.results, function(d) {
+          return null;
```

Comment:
What's the purpose of this change?

Reply:
It's to make it explicit that the return value doesn't matter